### PR TITLE
created server

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: routes/addImgSlider.routes.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        if (err) {
+            console.log(err)
+            res.status(500).send('An error occurred', err)
+        } else {
+            res.render('imagesPage', { items: items })
+        }
+    })
+})
+
+router.post('/addimage', upload.single('image'), (req, res, next) => {
```
Comment (vell28) [inline routes/addImgSlider.routes.js:30]: `add-image`. But would be better `create-image`

File: views/imagesPage.ejs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```
Comment (vell28) [inline views/imagesPage.ejs:1]: do we really need this file?

File: routes/addImgSlider.routes.js (hunk 1 above)
Comment (Ksenaaa) [reply routes/addImgSlider.routes.js:30]: fixed

File: views/imagesPage.ejs (hunk 2 above)
Comment (Ksenaaa) [reply views/imagesPage.ejs:1]: removed

**Review by vell28 — APPROVED** [review]

**Pull request merged**